chromadb/server/fastapi/__init__.py

```diff
@@ -324,10 +339,18 @@ def version(self) -> str:
             ),
         ),
     )
-    def create_database(
-        self, database: CreateDatabase, tenant: str = DEFAULT_TENANT
+    async def create_database(
+        self, request: Request, tenant: str = DEFAULT_TENANT
```

Comment:
Follow up: why do we take the `tenant` as a parameter alongside the `request`? Surely the `tenant` is part of the `request`, no?

Reply:
query or path params I leave as they are and use request to fetch body asynchronously